app/javascript/dashboard/i18n/locale/en/integrations.json

```diff
@@ -83,6 +83,28 @@
       "CREATE_ERROR": "There was an error creating a meeting link, please try again"
     },
     "OPEN_AI": {
+      "AI_ASSIST": "AI Assist",
+      "WITH_AI": "With AI",
```

Comment:
Using sentence case here, also maybe we should use interpolation here if possible, some languages may have the operator and noun before the subject.